iosApp/iosApp/Localizable.xcstrings

```diff
@@ -2908,6 +2908,9 @@
           }
         }
       }
+    },
+    "Add Stops" : {
+
```

Comment:
If this doesn’t go away with an Xcode build, then something is still referencing it. I’d check myself, but I’m still very deep in route branching work.